BUILDING.md

```diff
@@ -9,20 +9,74 @@ If you consistently can reproduce a test failure, search for it in the
 file a new issue.
 
 
+## Supported platforms
+
+This list of supported platforms is current as of the branch / release to which it is attached.
+
+### Input
+
+We rely on a few dependencies that _makes us who we are—most importantly V8 and libuv. We therefore need to adopt their supported platforms and potentially add to their lists based on test and/or release coverage.
```

Comment:
Micro nits
- `_makes` -> `make` (remove underscore, go singular)
- `who we are-most` -> `who we are - most` (spacing)